test/packager.test.ts

```diff
@@ -111,6 +111,31 @@ test("overwrites the output object if an option is provided", async () => {
   expect(data).toEqual("updated output\n");
 });
 
+test("recovers old inputs if output did not succeed for a given date", async () => {
```

Comment:
suggestion: can you also add a test for the case where the processed file already exists?